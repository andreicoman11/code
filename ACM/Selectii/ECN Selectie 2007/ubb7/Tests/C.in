100
1 -3 3
2 -1 1
-2 -3 3
-1 -3 2
-3 -2 2
2 0 3
1 3 2
1 -3 0
0 2 3
-3 -1 0
2 1 0
2 -1 3
1 0 1
3 3 2
-2 -3 3
1 3 1
1 -3 3
2 3 1
-2 2 0
-3 -3 1
3 -1 2
-2 -2 1
-1 1 3
-2 3 2
-2 -3 3
1 2 3
3 -3 2
-1 3 2
-2 3 0
-1 3 3
2 3 0
-3 -2 1
-2 0 3
3 -2 0
2 2 0
1 -3 2
1 3 2
2 3 3
1 2 3
-1 3 0
2 -2 1
1 1 2
2 -1 2
2 3 2
0 1 1
2 3 1
2 -1 0
2 -1 0
0 -3 2
-2 -3 3
-508995 -409992 323685
553392 -483251 728542
502221 817547 93040
562513 176009 256758
-751200 164156 849087
396544 -206605 725502
-305232 -8391 184602
-947054 -287758 694492
9686 -148989 549996
593263 950793 338533
830412 744271 164950
-584428 169209 854173
143317 14215 612994
-438671 -203900 943040
-253049 -875714 747054
-462892 103894 149380
639099 -205211 562427
305175 -42528 807444
424211 849402 164729
-778654 876372 467940
-272731 426782 744903
-956313 289781 925627
-778140 298273 813841
-380993 -780301 751473
549892 -137200 14403
-945781 759849 796192
-790985 100474 848910
-896861 -24883 819834
-736003 699278 406672
-891043 614101 229970
-448175 -407858 637201
-404634 482985 468605
88215 729271 501493
336212 174043 164316
188270 -240801 363585
521148 176338 81702
-841404 310818 49528
931540 -347279 595380
-605958 -918773 746344
-952329 -374347 464129
-931187 -27779 992989
386248 -702962 947143
-248703 548611 362237
-986215 570895 394558
-9136 807452 477133
906897 107191 835776
-845615 -37511 676833
-828796 -49281 517683
978941 -426668 798656
128896 -271438 628044
