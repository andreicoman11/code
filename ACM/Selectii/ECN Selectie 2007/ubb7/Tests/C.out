do not advertise
do not advertise
do not advertise
do not advertise
do not advertise
do not advertise
does not matter
do not advertise
do not advertise
advertise
do not advertise
do not advertise
do not advertise
do not advertise
do not advertise
advertise
do not advertise
does not matter
advertise
do not advertise
do not advertise
do not advertise
do not advertise
advertise
do not advertise
do not advertise
do not advertise
advertise
advertise
advertise
advertise
does not matter
do not advertise
do not advertise
does not matter
do not advertise
does not matter
do not advertise
do not advertise
advertise
do not advertise
do not advertise
do not advertise
do not advertise
does not matter
does not matter
do not advertise
do not advertise
do not advertise
do not advertise
do not advertise
do not advertise
advertise
do not advertise
advertise
do not advertise
advertise
do not advertise
do not advertise
advertise
do not advertise
do not advertise
do not advertise
do not advertise
do not advertise
advertise
do not advertise
do not advertise
advertise
advertise
do not advertise
advertise
advertise
do not advertise
do not advertise
advertise
advertise
advertise
advertise
advertise
do not advertise
advertise
advertise
do not advertise
do not advertise
do not advertise
advertise
do not advertise
do not advertise
advertise
do not advertise
do not advertise
advertise
advertise
advertise
do not advertise
advertise
advertise
do not advertise
do not advertise
